Multisig/UI/App/AppDelegate.swift

```diff
@@ -40,4 +42,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // If any sessions were discarded while the application was not running, this will be called shortly after application:didFinishLaunchingWithOptions.
         // Use this method to release any resources that were specific to the discarded scenes, as they will not return.
     }
+
+    func application(_ application: UIApplication, didFailToRegisterForRemoteNotificationsWithError error: Error) {
```

Comment:
I would prefer to move this func to AppDelegate+Messaging.swift so we have everything related to notifications there

Reply:
This is not possible to move completely, because it is implementation of the UIApplicationDelegate protocol, it has to be here

Reply:
true, my mistake